bot/cogs/random_cog/random_cog.py

```diff
@@ -133,5 +134,32 @@ async def slotsrolling(input, spinstatus,waittime):
         embed = await slotsrolling(f'{a} | {b} | {c}', f'**{message}**',1.75)
         await msg.edit(embed = embed)
 
+    @commands.command()
+    async def raffle(self, ctx, time: typing.Optional[int] = 5, *, reason):
```

Comment:
For time we actually have a custom TimeDuration converter to allow for things like `!raffle 20m some reason` that should be used here to make this command super flexible